<launch>
  <param name="/camera/driver/depth_registration" value="True" />
  
  <node pkg="face_tracker" name="object_tracker" type="object_tracker.py" output="screen">
  
    <remap from="camera_info" to="/camera/rgb/camera_info" />
    <remap from="depth_image" to="/camera/depth/image" />

    <rosparam>
       rate: 10
       max_z: 2.0
       goal_z: 0.6
       z_threshold: 0.05
       x_threshold: 0.1
       z_scale: 1.0
       x_scale: 2.0
       max_rotation_speed: 0.2
       min_rotation_speed: 0.02
       max_linear_speed: 0.7
       min_linear_speed: 0.2
    </rosparam>
    
  </node>
</launch>
